src/entity/Song.java

```diff
@@ -7,22 +7,22 @@ public class Song extends DataObject{
     private final int length;
     private List<String> artists = new ArrayList<>();
 
-    private final String album;
+    private final Album[] albums;
 
-    public Song(String id, String name, int length, List<String> artists, String album){
+    public Song(String id, String name, int length, List<String> artists, Album[] albums){
         super(id, name);
         this.length = length;
         this.artists = artists;
-        this.album = album;
+        this.albums = albums;
     }
 
 
     public List<String> getArtist() {
         return artists;
     }
 
-    public String getAlbum() {
-        return album;
+    public Album[] getAlbums() {
```

Comment:
here is Album a list, maybe try rechecking I think it might be a String that youre returning